### DISCUSSION TITLE
Chia won't start up after crash during plot

### DISCUSSION THREAD

**Discussion by dreadwoodgames** [Chia-Network/chia-blockchain#4763] [Support]
My computer crashed while parallel plotting 2 plots and on reboot Chia will not start up, it is stuck at "Connecting to Wallet". I have tried force restarting the app, running it from the CLI and rebooting my computer. After all that did nothing I tried deleting the temp files on my plot drive (SSD) and tried the above with no success. Any help would be much appreciated!

**Comment by dreadwoodgames** [comment]
Update: Chia did eventually reconnect to wallet after 15-20 min. This thread can be closed.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
